cypress/plugins/testDataProcessor.js

```diff
@@ -0,0 +1,390 @@
+const fs = require('fs');
+const jsonMerger = require('json-merger');
+const CONSTANTS = require('../support/constants/constants');
+const REGEXFORMATS = require('../support/constants/regexformats');
+let envVariables;
+const path = require('path');
+const _ = require('lodash');
+const logger = require('../support/Logger')('testDataProcessor.js');
+
+function testDataProcessor(configEnv) {
+  envVariables = configEnv;
+  const fcsFireboltCalls = CONSTANTS.FIREBOLTCALLS_FROM_FCS;
+  const configModuleFireboltCalls = CONSTANTS.FIREBOLTCALLS_FROM_CONFIGMODULE;
+  const fcsFireboltMocks = CONSTANTS.FIREBOLTMOCK_FROM_FCS;
+  const configModuleFireboltMocks = CONSTANTS.FIREBOLTMOCK_FROM_CONFIGMODULE;
+
+  // Merging all JSON files from the directory.
+  const fcsFireboltCallsMergedJson = fetchMergedJsonFromDirectory(fcsFireboltCalls);
+  const configFireboltCallsModuleMergedJson =
+    fetchMergedJsonFromDirectory(configModuleFireboltCalls);
+
+  const fcsFireboltMocksMergedJson = fetchMergedJsonFromDirectory(fcsFireboltMocks);
+  const configModuleFireboltMocksMergedJson =
+    fetchMergedJsonFromDirectory(configModuleFireboltMocks);
+
+  // Combining the FCS and config module JSON Data
+  const combinedFireboltCallsJson = Object.assign(
+    fcsFireboltCallsMergedJson,
+    configFireboltCallsModuleMergedJson
+  );
+  const combinedFireboltMocksJson = Object.assign(
+    fcsFireboltMocksMergedJson,
+    configModuleFireboltMocksMergedJson
+  );
+
```

Comment:
ConfigModuleSetResponseJson to be changed to configModuleSetResponseJson

Reply:
addressed the comment